src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClientStatus.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Azure.Data.AppConfiguration;
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class ConfigurationClientStatus
```

Comment:
I will try again :)

This class is really a wrapper of the client. The client instance, endpoints, status etc. are just the properties of the client. You even name the variable of this class as "client" in ConfigurationClientManager. Suggest to change to `ConfigurationClientWrapper`.

Reply:
I honestly don't quite like `ConfigurationClientWrapper`, but I think we can go with it because of lack of a better alternative.